packages/roosterjs-editor-api/lib/format/insertImage.ts

```diff
@@ -1,6 +1,11 @@
 import { Editor } from 'roosterjs-editor-core';
 import { PluginEvent, PluginEventType } from 'roosterjs-editor-types';
 
+/**
+ * Insert an image to editor
+ * @param editor The editor instance
+ * @param imageFile The image file
```

Comment:
Explain how to obtain the file object: (at least 3 ways: From local file, from clipboard data, from drag-and-drop)